classes/Image.php

```diff
@@ -24,6 +24,7 @@
  * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
  */
 
+use PrestaShop\PrestaShop\Core\Exception\InvalidArgumentException;
 /**
```

Comment:
cs fixer should probably be mad
```suggestion

/**
```